sqldelight-gradle-plugin/build.gradle

```diff
@@ -46,6 +46,11 @@ dependencies {
 }
 
 test {
+  inputs.files("../gradle/dependencies.gradle").withPathSensitivity(PathSensitivity.RELATIVE)
+  inputs.property("ANDROID_HOME", System.getenv("ANDROID_HOME"))
```

Comment:
This will prevent cache relocation, as you are using an absolute path. I'm sure you don't care about the path at all, but about what's installed there.